bin/src/ctl/command.rs

```diff
@@ -10,7 +10,7 @@ use std::{
 use anyhow::{self, bail, Context};
 use prettytable::Table;
 use rand::{distributions::Alphanumeric, thread_rng, Rng};
-
+use serde::Serialize;
```

Comment:
we need an empty line between extern crate use statements, and workpsace use statements.